### PR TITLE
Fixes #11180: Package filter rules edit checked for wrong type

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Change looks good, but before we merge I should check to make sure we didn't forget to 'update' existing filters.

**Comment by jlsherrill** [comment]
Oh actually this is just checking constants, so no update needed.  One comment though

File: app/controllers/katello/api/v2/content_view_filter_rules_controller.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def show
     def update
       update_params = rule_params
 
-      if @rule.filter.content_type == 'package'
+      if @rule.filter.content_type == 'rpm'
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/content_view_filter_rules_controller.rb]: Can we just reference  ContentViewPackageFilter::CONTENT_TYPE here ?

**Comment by ehelms** [comment]
Updated to the suggestion

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**